go-controller/pkg/cni/cni.go

```diff
@@ -65,22 +64,20 @@ func podDescription(pr *PodRequest) string {
 	return fmt.Sprintf("[%s/%s]", pr.PodNamespace, pr.PodName)
 }
 
-func (pr *PodRequest) cmdAdd(kclient kubernetes.Interface) ([]byte, error) {
+func (pr *PodRequest) cmdAdd(podLister corev1listers.PodLister) ([]byte, error) {
 	namespace := pr.PodNamespace
 	podName := pr.PodName
 	if namespace == "" || podName == "" {
 		return nil, fmt.Errorf("required CNI variable missing")
 	}
 
-	kubecli := &kube.Kube{KClient: kclient}
-
 	// Get the IP address and MAC address from the API server.
```

Comment:
Consider updating the comment now that this info is coming from the cache and not doing a query to the API server.

Reply:
yeah